wpilib/wpilib/controllerpower.py

```diff
@@ -29,7 +29,7 @@ def getInputVoltage():
         return hal.getVinVoltage()
```

Comment:
Please remove the extraneous return type documentation. (I seem to keep saying this, and there's a lot in here so… just imagine a million dittos.)